tensorflow/lite/kernels/internal/BUILD

```diff
@@ -141,38 +141,36 @@ cc_library(
 cc_library(
     name = "portable_tensor_utils",
     srcs = [
-        "reference/portable_tensor_utils.cc",
     ],
     hdrs = [
-        "reference/portable_tensor_utils.h",
-        "reference/portable_tensor_utils_impl.h",
     ],
     copts = tflite_copts(),
     deps = [
         ":common",
         ":compatibility",
         ":cppmath",
-        "@gemmlowp",
     ],
 )
```

Comment:
since the hdrs are srcs are empty, this entire BUILD target can be removed.

Reply:
I didn't realize I forgot to delete it , done!